### ISSUE TITLE
General NestedTensor op coverage tracking issue

### ISSUE THREAD

**Issue by drisspg** [pytorch/pytorch#83994]
### This issue is to have a centralized place to list and track work on adding support to new ops for the NestedTensor backend.
There are  large number of operators in pytorch and so they are not all implemented yet for the NestedTensor backends as it is still in the prototype phase. We will be prioritizing adding new operators based on user feedback. If possible, please also provide link to the network or use-case where this op is getting used.

If you want to work on adding support for such op, feel free to comment below to get assigned one. Please avoid picking up an op that is already being worked on or that already has a PR associated with it.

#### Op coverage requests
- [ ] aten::convolution
- [ ] aten::max.dim
- [x] detach #84078 
- [x] to #87146
- [ ] eq
- [ ] masked_select
- [ ] index_select
- [ ] narrow
- [ ] alias
- [ ] Broadcasting Ufuncs along implicit NT dim
- [ ] Zero-copy Nt construction from Size info
- [ ] BCE/ Other loss functions for NT
- [ ] nested tensor creation from arbitrary masks rather than left-aligned

#### Backward op coverage requests
- [ ] gelu, relu backward
- [ ] layernorm backward

cc @cpuhrsch @jbschlosser @bhosmer

**Comment by drisspg** [comment]
#88137

**Comment by vadimkantorov** [comment]
also, having clean nestedtensor support for depthwise conv is important for Conformer models which have them in convolution blocks
